src/main/java/org/jabref/gui/commonfxcontrols/FieldFormatterCleanupsPanelViewModel.java

```diff
@@ -28,6 +28,12 @@
     private final ObjectProperty<Formatter> selectedFormatterProperty = new SimpleObjectProperty<>();
 
     public FieldFormatterCleanupsPanelViewModel() {
+        availableFieldsProperty.sort((field1, field2) -> {
+            if (field1.getDisplayName().equals(field2.getDisplayName())) {
```

Comment:
The compareTo already performs equal checks...
So you should be able to directly use sort(field1, field2) -> field1.getDisplayName().compareTo(field2.getDisplayName))

Reply:
`Comparator.comparing` should work as well.

Reply:
What you said really makes sense and I have modified them.